packages/indexer-client/src/types/VertexTx.ts

```diff
@@ -11,7 +11,10 @@ export interface VertexLiquidateSubaccountTx {
     sender: string;
     liquidatee: string;
     mode: number;
-    health_group: number;
+    // On V2 - should encode health group
+    product_id: number | undefined;
```

Comment:
Also - confirming that these are still u32 and coming back as numbers and not strings?

Reply:
will confirm!

Reply:
it's a number

Reply:
Okay this should be good for now then 👍🏻 but any other fields that should be added / removed?